app/views/messages/_form.html.erb

```diff
@@ -13,8 +13,6 @@
 
 <div class="absolute bottom-0 inset-x-0 pl-3 pr-2 py-2 flex justify-end">
   <div class="shrink-0">
-    <button type="submit" class="inline-flex items-center px-4 py-2 border border-transparent text-sm font-medium rounded-md shadow-sm text-white bg-gray-600 hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500">
```

Comment:
Ah, see, this one doesn't use `ml-3`. Even though it might not make a visual difference here, I'd rather pull it out and have the others pass it in.

Reply:
Good catch! I didn't noticed that.